core/go/internal/transportmgr/reliable_msg_handler.go

```diff
@@ -33,13 +33,15 @@ import (
 )
 
 const (
-	RMHMessageTypeAck                 = "ack"
-	RMHMessageTypeNack                = "nack"
-	RMHMessageTypeStateDistribution   = string(pldapi.RMTState)
-	RMHMessageTypeReceipt             = string(pldapi.RMTReceipt)
-	RMHMessageTypePreparedTransaction = string(pldapi.RMTPreparedTransaction)
-	RMHMessageTypePrivacyGroup        = string(pldapi.RMTPrivacyGroup)
-	RMHMessageTypePrivacyGroupMessage = string(pldapi.RMTPrivacyGroupMessage)
+	RMHMessageTypeAck                         = "ack"
+	RMHMessageTypeNack                        = "nack"
+	RMHMessageTypeStateDistribution           = string(pldapi.RMTState)
+	RMHMessageTypeReceipt                     = string(pldapi.RMTReceipt)
+	RMHMessageTypePublicTransaction           = string(pldapi.RMTPublicTransaction)
+	RMHMessageTypePublicTransactionSubmission = string(pldapi.RMTPublicTransactionSubmission)
```

Comment:
Noting few places in the comments to talk about these - with an action to lay out the model (public arch docs).